CommonLibSF/CMakeLists.txt

```diff
@@ -0,0 +1,166 @@
+cmake_minimum_required(VERSION 3.21)
+
+# singleton target across multiple projects
+if(TARGET CommonLibSF)
+	return() 
+endif()
+
+# options if not defined
+option(SFSE_SUPPORT_XBYAK "Enables trampoline support for Xbyak." ON)
+
+# standards & flags
+set(CMAKE_CXX_STANDARD 23)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION ON)
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_DEBUF OFF)
```

Comment:
typo

Reply:
I actually have never found that one, now it's surely a debuf